deployer/templates/hawkular-metrics.yaml

```diff
@@ -97,9 +102,7 @@ objects:
           - "--hmw.truststore=/secrets/hawkular-metrics.truststore"
           - "--hmw.keystore_password_file=/secrets/hawkular-metrics.keystore.password"
           - "--hmw.truststore_password_file=/secrets/hawkular-metrics.truststore.password"
-          - "--hmw.jgroups_keystore=/secrets/hawkular-metrics.jgroups.keystore"
-          - "--hmw.jgroups_keystore_password_file=/secrets/hawkular-metrics.jgroups.keystore.password"
-          - "--hmw.jgroups_alias_file=/secrets/hawkular-metrics.jgroups.alias"
+          - "-DJGROUPS_PASSWORD=${JGROUPS_PASSWORD}"
```

Comment:
Please do not pass the password as an parameter like this, it will be visible to other users via things like ps